tests/unit/recommendation.test.js

```diff
@@ -1,75 +0,0 @@
-/* eslint-disable no-undef */
-import * as recommendationService from '../../src/services/recommendationService.js';
-import * as recommendationRepository from '../../src/repository/recommendationRepository.js';
-
-const sut = recommendationService;
-
-const mockRepository = {
-  createRecommendation: () => jest.spyOn(recommendationRepository, 'createRecommendation'),
-  checkForExistentRec: () => jest.spyOn(recommendationRepository, 'checkForExistentRec'),
-  addPoint: () => jest.spyOn(recommendationRepository, 'addPoint'),
-  dropPoint: () => jest.spyOn(recommendationRepository, 'dropPoint'),
-  getRandomRec: () => jest.spyOn(recommendationRepository, 'getRandomRec'),
-  getTopRec: () => jest.spyOn(recommendationRepository, 'getTopRec'),
-};
-
-describe('Post tests for recommendations', () => {
-  beforeEach(() => {
-    mockRepository.createRecommendation().mockReset();
-    mockRepository.checkForExistentRec().mockReset();
-    mockRepository.addPoint().mockReset();
-    mockRepository.dropPoint().mockReset();
-    mockRepository.getRandomRec().mockReset();
-    mockRepository.getTopRec().mockReset();
-  });
-
-  it('Return correct body for valid recommendation', async () => {
-    const body = {
```

Comment:
seria interessante manter um objeto default no topo do arquivo e dar spread quando for utilizar algo diferente do default